eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -173,12 +173,11 @@
           "code": "java.class.nonPublicPartOfAPI",
           "new": "(class|enum|interface) org\\.conscrypt\\..*",
           "justification": "skip check for third party files."
-        },
-        {
+        },        {
```

Comment:
nit, json format

Is this done by agent? I saw sometimes agent do this.

Reply:
Yes, it's done by agent :)

Reply:
fixed